### PR TITLE
Scroll up the feed screen when press a tab

### LINKED ISSUE #113: Scroll up when you press a tab
## Kind (Required)

- Improvement

 ## Overview (Required)

- It seems that we need to communicate the event well

 ## Links	

-

### PR DESCRIPTION
**Pull request by t-yusaku**
## Issue
- close #113 

## Overview (Required)
Now, we can scroll up the feed screen when we press a tab
- add `LazyListState` for the feed list
- use `LazyListState.animateScrollToItem`

## Links
- https://developer.android.com/reference/kotlin/androidx/compose/foundation/lazy/LazyListState#animatescrolltoitem
- https://developer.android.com/jetpack/compose/lists#control-scroll-position
- https://developer.android.com/jetpack/compose/kotlin#coroutines

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

File: uicomponent-compose/feed/src/main/java/io/github/droidkaigi/feeder/feed/FeedScreen.kt [hunk 1]
```diff
@@ -100,6 +106,9 @@ fun FeedScreen(
         filters = state.filters,
         onSelectTab = { tab: FeedTabs ->
             selectedTab = tab
+            coroutineScope.launch {
+                listState.animateScrollToItem(index = 0)
```
Comment (takahirom) [inline uicomponent-compose/feed/src/main/java/io/github/droidkaigi/feeder/feed/FeedScreen.kt:110]: I didn't know  this api!✨

**Review by takahirom — COMMENTED** [review]
Can you fix conflict?

**Comment by t-yusaku** [comment]
Yes. I will fix it!

File: uicomponent-compose/feed/src/main/java/io/github/droidkaigi/feeder/feed/FeedScreen.kt (hunk 1 above)
Comment (t-yusaku) [reply uicomponent-compose/feed/src/main/java/io/github/droidkaigi/feeder/feed/FeedScreen.kt:110]: I knew this api from your comment 😊 
https://github.com/DroidKaigi/conference-app-2021/issues/113#issuecomment-787013964

btw, I noticed that `snapToItemIndex` was renamed to `scrollToItem` 😲

> ScrollableState.smoothScrollBy() was renamed to animateScrollBy() LazyListState.snapToItemIndex() was renamed to scrollToItem() ScrollState.smoothScrollTo() was renamed to animateScrollTo() (I35ded)

https://developer.android.com/jetpack/androidx/releases/compose-foundation

**Comment by tomoya0x00** [comment]
@t-yusaku 
I fixed additional conflict 👍  But still remains some check warnings 😇 
Could you solve it? Thanks!

**Comment by tomoya0x00** [comment]
@t-yusaku 
It seems you can just run ` ./gradlew :spotlessApply`, and solve check warnings.

```
* What went wrong:
Execution failed for task ':spotlessKotlinCheck'.
> The following files had format violations:
      uicomponent-compose/feed/src/main/java/io/github/droidkaigi/feeder/feed/FeedScreen.kt
          @@ -39,8 +39,8 @@

           import·io.github.droidkaigi.feeder.core.theme.ConferenceAppFeederTheme
Deprecated Gradle features were used in this build, making it incompatible with Gradle 7.0.
           import·io.github.droidkaigi.feeder.core.use
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/6.8.2/userguide/command_line_interface.html#sec:command_line_warnings
           import·io.github.droidkaigi.feeder.core.util.collectInLaunchedEffect
          +import·kotlin.reflect.KClass
           import·kotlinx.coroutines.launch
          -import·kotlin.reflect.KClass
           
           sealed·class·FeedTabs(val·name:·String,·val·routePath:·String)·{
           ····object·Home·:·FeedTabs("Home",·"home")
  Run './gradlew :spotlessApply' to fix these violations.
```

**Comment by t-yusaku** [comment]
@tomoya0x00 
Sorry to being late 💦 
I fixed it! 😊

**Review by takahirom — APPROVED** [review]
LGTM 👍

**Pull request merged**